src/apps/loan-list/materials/stackable-material/material-status.tsx

```diff
@@ -32,7 +27,13 @@ const MaterialStatus: FC<MaterialStatusProps> = ({
             warningText={t("loanListStatusBadgeWarningText")}
           />
           <p className="text-small-caption" id="due-date">
-            {dueDateLabel} {formatDate(dueDate)}
+            {isDigital(loan)
+              ? t("loanListToBeDeliveredDigitalMaterialText", {
+                  placeholders: { "@count": formatDate(dueDate) }
+                })
+              : t("loanListToBeDeliveredText", {
+                  placeholders: { "@count": formatDate(dueDate) }
+                })}
```

Comment:
You could also make a function to get the text, it is almost similar.
But you are also pointing `@count`to a date here. I really do not get it :)